File: Labs/Lab2/questionOne.java
package Lab2;
import java.util.Arrays;


public class questionOne {

    public static boolean isAnagram(String s, String t) {
       
        char[] sArray = s.toCharArray();
        char[] tArray = t.toCharArray();
        
        
        Arrays.sort(sArray);
        Arrays.sort(tArray);
        
       
        return Arrays.equals(sArray, tArray);
    }

    public static void main(String[] args) {
        String s = "anagram";
        String t = "nagaram";
        System.out.println(isAnagram(s, t));  
        
        String s2 = "rat";
        String t2 = "car";
        System.out.println(isAnagram(s2, t2));
    }
    
}

File: Labs/Lab2/questionOne.py
def  anagram_checker(s,t):
    return sorted(s) == sorted(t)

if __name__ == "__main__":
    s = "anagram"
    t = "nagaram"
    print(anagram_checker(s,t))

    s2 = "rat"
    t2 = "car"
    print(anagram_checker(s2, t2))
